Mathlib/Order/GaloisConnection.lean

```diff
@@ -98,10 +98,12 @@ theorem monotone_u : Monotone u := fun a _ H => gc.le_u ((gc.l_u_le a).trans H)
 theorem monotone_l : Monotone l :=
   gc.dual.monotone_u.dual
 
-theorem upperBounds_l_image (s : Set α) : upperBounds (l '' s) = u ⁻¹' upperBounds s :=
+theorem upperBounds_l_image (gc : GaloisConnection l u) (s : Set α) :
```

Comment:
Same question: delete the `gc` hypothesis 31 lines above?

Reply:
As above.